lesson_2/cmd/app/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"../../pkg/fibonacci"
+	"fmt"
+)
+
+func main() {
+	var n int
+
+	defer fmt.Println("\nCompleted!")
+	fmt.Print("This program can count fibonacci numbers\n", "Please enter number: ")
+	fmt.Scan(&n)
+
+
+	fmt.Println("Count by with liner execution time:")
+	for i := 1; i < n; i++ {
+		fmt.Print(fibonacci.FibLoop(i)," ") 	//by loop
```

Comment:
fibonacci.FibLoop() Do we relly need Fib in function name? Seems Loop to be enough.